i18n/en.json

```diff
@@ -1825,11 +1825,11 @@
   },
   {
     "id": "api.team.import_team.unavailable.app_error",
-    "translation": "Filesize unavilable"
+    "translation": "Filesize unavailable"
```

Comment:
This is for slack import so @grundleborg might be better apt to answer but maybe "Filesize not included" would be better since the error occurs when the filesize parameter is missing